week-1-ts-ecosystem/w01e03-object-oriented-structure/src/model/cart.ts

```diff
@@ -0,0 +1,71 @@
+import { v4 } from 'uuid';
+import { CartType, Currency, ICartItem } from './cart.model';
+
+export class CartItem implements ICartItem {
```

Comment:
Wyprowadziłbym to do innego - osobnego pliku. Żeby ten model też od razu widzieć.